apps/etterlatte-trygdetid/src/main/kotlin/trygdetid/regler/BeregnTrygdetid.kt

```diff
@@ -186,6 +171,45 @@ val fremtidigTrygdetid =
         }
     }
 
+/**
+ * Finn første dato for fremtidig trygdetid
+ */
+val fremtidigTrygdetidFra =
+    RegelMeta(
+        gjelderFra = TRYGDETID_DATO,
+        beskrivelse = "Henter ut periode for fremtidig trygdetid",
+        regelReferanse = RegelReferanse(id = "REGEL-FINN-FREMTIDIG-TRYGDETIDSPERIODE"),
+    ) benytter normalisertTrygdetidGrunnlagListe med { trygdetidPerioder ->
+        trygdetidPerioder.firstOrNull { it.type == TrygdetidType.FREMTIDIG }?.let {
```

Comment:
Er listen med trygdetidPerioder garantert sortert på `it.periode.fra` og/eller er det garantert kun <= fremtidig trygdetid? I tilfelle to lurer jeg på om en `singleOrNull` er bedre i regelen, og sikkert ha noe validering i service. I tilfelle en tenker jeg en kommentar om at de er garantert sortert er rimelig.

I tilfelle tre, der vi ikke har de garantiene vil du vel kunne få feil i perioden du henter ut -- en sortering først er nok rimelig

Reply:
Den regelen er kun flyttet. Den er både sortert og det skal kun være en. Men jeg kan endre til single.